src/passes/MultiMemoryLowering.cpp

```diff
@@ -115,100 +115,174 @@ struct MultiMemoryLowering : public Pass {
       replaceCurrent(builder.makeCall(funcName, {}, curr->type));
     }
 
+    Expression* makeGtuTrap(Expression* leftOperand,
```

Comment:
It might be good to add "Add" to the name because it adds before doing the comparison.
```suggestion
    Expression* makeAddGtuTrap(Expression* leftOperand,
```

Reply:
Makes sense, thanks!